File: SecondoAnno/Programmazione_II/Java/ESAMI/Esame24Gennaio2020/Java/Campo.java
package Esame24Gennaio2020.Java;

public class Campo {

    protected int codice, prenotazione;
    protected String campo;
    protected float larghezza, lunghezza, costoOrario;

    public Campo(int codice, String campo, float larghezza, float lunghezza, float costoOrario) {
        this.codice = codice;
        this.campo = campo;
        this.larghezza = larghezza;
        this.lunghezza = lunghezza;
        this.costoOrario = costoOrario;
    }

    public void addPrenotazione() {
        prenotazione++;
    }

    public float incasso() {
        return prenotazione*costoOrario;
    }
}

File: SecondoAnno/Programmazione_II/Java/ESAMI/Esame24Gennaio2020/Python/Campo.py
class Campo:
    
    def __init__(self, codice, campo, larghezza, lunghezza, costoOrario):
        super.codice = codice
        super.campo = campo
        self.larghezza = larghezza
        self.lunghezza = lunghezza
        self.costoOrario = costoOrario
        self.prenotazioni = 0

    def addPrenotazione(self):
        self.prenotazioni = self.prenotazioni + 1

    def incasso(self) -> float:
        return self.prenotazioni * self.costoOrario
    
    def __str__(self) -> str:
        return self.campo + "\t" + str(self.codice) + "\t" + str(self.larghezza) + "\t" + str(self.lunghezza) + "\t" + str(self.costoOrario)